pywt/_extensions/c/common.h

```diff
@@ -23,14 +23,14 @@
         #include "Python.h"
     #endif
 
-    typedef Py_ssize_t index_t;
+    typedef Py_ssize_t pywt_index_t;
```

Comment:
If we won't have CI then put a comment in here mentioning SmartOS/Solaris at the reason for namespacing.